crates/bevy_pbr/src/light_probe/generate.rs

```diff
@@ -621,6 +755,27 @@ fn create_storage_view(texture: &Texture, mip: u32, _render_device: &RenderDevic
     })
 }
 
+/// To ensure compatibility in web browsers, each call returns a unique resource so that multiple missing mip
```

Comment:
I swear this was never a problem on native. Maybe a web-only validation? Idk

Reply:
This is the only trail of info I found on the matter
https://github.com/gpuweb/gpuweb/issues/287

But it shows up as a warning in the browser console and crashes the pipeline unless I create unique resources for each binding.